src/components/Routes.js

```diff
@@ -29,6 +30,7 @@ const Routes = () => (
     <Route exact path="/search" component={Search} />
     <Route exact path="/login" component={Login} />
     <Route exact path="/articles/create" component={CreateArticles} />
+    <Route exact path="/articles/mine" component={SpecificUserArticles} />
```

Comment:
Particularly for the user to get all his articles you have given me the feedback to name the route to ` articles/mine ` for your information I don't really need this component now because I have had to pass it in the profile page,If I name it articles it will have the route will retrieve all the articles without in the database

Reply:
cool